pkgs/games/openarena/default.nix

```diff
@@ -1,43 +1,96 @@
-{ lib, fetchurl, makeWrapper, patchelf, pkgs, stdenv, SDL, libglvnd, libogg, libvorbis, curl, openal }:
+{ copyDesktopItems
+, autoPatchelfHook
+, curl
+, fetchzip
+, lib
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md the very last line:
"Arguments should be listed in the order they are used, with the exception of lib, which always goes first."

Shouldn't lib be the first argument ?

Reply:
Done.